org.eclipse.dartboard/src/org/eclipse/dartboard/pub/PubGetHandler.java

```diff
@@ -0,0 +1,36 @@
+package org.eclipse.dartboard.pub;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.Adapters;
+import org.eclipse.core.runtime.preferences.InstanceScope;
+import org.eclipse.dartboard.Constants;
+import org.eclipse.jface.viewers.ISelection;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.ui.handlers.HandlerUtil;
+import org.eclipse.ui.preferences.ScopedPreferenceStore;
+
+public class PubGetHandler extends AbstractHandler {
+
+	private PubService pub;
+
+	private ScopedPreferenceStore preferences;
+
+	public PubGetHandler() {
+		pub = PubService.getInstance();
+		preferences = new ScopedPreferenceStore(InstanceScope.INSTANCE, Constants.PLUGIN_ID);
```

Comment:
I think it's not the right place to create the object.  Is the preference store object is not available at this point? If not, we need to create in startup class and need to access like activator

  ```
	public IPreferenceStore getPreferenceStore() {
		if (preferenceStore == null) {
			preferenceStore = new ScopedPreferenceStore(InstanceScope.INSTANCE, Constants.PLUGIN_ID);

		}
		return preferenceStore;
	}
  
```

Reply:
I'm not sure what the benefit of a globally accessible constant/variable would be. Again, it would be nice to have it as an `@Inject`'able service, but I have not figured out how to do this. I will have to wait for @vogella for advice on this.

Also it is done this way all over the codebase right now and would require a major refactor that is IMO out of scope of this PR.